metrics/bin/metrics.dart

```diff
@@ -0,0 +1,163 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:crypto/crypto.dart';
+import 'package:logging/logging.dart';
+import 'package:metrics/metrics.dart';
+import 'package:metrics/src/options.dart';
+import 'package:path/path.dart' as path;
+import 'package:pubspec_parse/pubspec_parse.dart';
+
+Future<void> main(List<String> arguments) async {
+  late Options options;
+  bool displayUsage = false;
+  try {
+    options = parseOptions(arguments);
+    displayUsage = options.help;
+  } catch (e) {
+    displayUsage = true;
+    exitCode = 64; // EX_USAGE (by convention, see sysexits.h)
```

Comment:
Should we set `exitCode` here? This conflicts with the comment at the end of the function that we don't want to cause broken builds.

Reply:
I think this is different, as it pertains to the user giving wrong inputs. 
The comment in the bottom is more for network errors, etc.

Reply:
True, but I don't suppose users will be running this script ever - it's all part of our automated scripts. In any case, I don't have strong opinions on the subject, just found it surprising as I was reading the code.

Reply:
True. I will try to make the comments clearer.